domains/ven.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "Ven Domain",
+  "repo": "https://github.com/jayzrc/Vencordy",
+  "owner": {
+    "username": "jayzrc",
+  },
+  "record": {
+    "A": "217.182.75.40"
```

I cannot reach your website, 404
Also A record should be an array of IPs
```suggestion
    "A": ["217.182.75.40"]
```